test/src/main/java/org/axonframework/test/aggregate/AggregateTestFixture.java

```diff
@@ -495,7 +508,8 @@ private void registerAggregateCommandHandlers() {
     private void ensureRepositoryConfiguration() {
         if (repository == null) {
             registerRepository(EventSourcingRepository.builder(aggregateType)
-                                                      .aggregateFactory(new GenericAggregateFactory<>(aggregateType))
+                                                      .aggregateModel(aggregateModel())
```

Comment:
aggregateModel() is invoked twice. Consider invoking it once and passing the result twice to both methods.